src/DistributedComputations/distributed_fft_based_poisson_solver.jl

```diff
@@ -3,11 +3,12 @@ using PencilArrays: Permutation
 
 import FFTW 
 
+using Oceananigans.Solvers: AbstractSolver
 import Oceananigans.Solvers: poisson_eigenvalues, solve!
 import Oceananigans.Architectures: architecture
 
 
-struct DistributedFFTBasedPoissonSolver{P, F, L, λ, S, I}
+struct DistributedFFTBasedPoissonSolver{P, F, L, λ, S, I} <: AbstractSolver
```

Comment:
What's the point of AbstractSolver?

Reply:
It was a test IO was doing, I found a better way to solve the same problem